### DISCUSSION TITLE
Idea: Only run tests on frameworks that have changed between commits

### DISCUSSION THREAD

**Discussion by ReallySnazzy** [TechEmpower/FrameworkBenchmarks#9309] [Ideas]
The run times are getting quite large for the tests. Is there a reason why a commit hash can't be stored when the test runs and then on the next run only run the frameworks that have been modified in some way? Maybe after some time, the tests could be re-ran just to check the dockerfiles are still valid.

**Comment by volyrique** [comment]
Already been discussed in issues #5925 and #8185.
